index.html

```diff
@@ -28,6 +26,149 @@ <h2>I'am a Software developer</h2>
   <li><a href=""><i class="fa-solid fa-m"></i></a></li> 
   </ul>
   </section>
-  </main>
+  </section>
+  <section class="recent-work">
+  <div class="top">
+  <h4>My Recent Works</h4>
+  <img src="image/line.png" alt="">
+  </div>
+  <section class="fram-two">
+  <div class="featured-image">
+  <img src="image/Img-Placeholder.png" alt="">
+  </div>
+  <section class="right-block">
+  <h6>Multi-Post Stories</h6>
+  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+  <section class="tags">
+  <img src="image/css.png" alt="">
+  <img src="image/html.png" alt="">
+  <img src="image/bootstrap.png" alt="">
+  <img src="image/ruby.png" alt=""> <br>
+  <button>See Project</button>
```

Comment:
- [ ] (OPTIONAL) Kindly fix this section so that it looks like the Figma template. The heading is in the wrong position, it has the wrong background color, the button is not well-styled, the technology categories(CSS, HTML, Ruby) should be in a `ul` list and each of them an `li`.
- Your design:
![Annotation 2022-03-26 202538](https://user-images.githubusercontent.com/85113567/160254424-d93822de-0f9e-4282-b4a6-929fb1837b86.jpg)
- Figma design:
![Annotation 2022-03-26 202551](https://user-images.githubusercontent.com/85113567/160254432-5e3ff741-40d5-403d-8237-91de04db28bb.jpg)